bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/OverheadTablePrinter.java

```diff
@@ -109,7 +109,7 @@ private int getGsaoiLgsReacqNum() {
 
             if (cts != null) {
                 for (PlannedTime.CategorizedTime lct : cts) {
-                    if (lct.detail.equals("LGS Reacquisition")) {
+                    if (lct.detail.contains("LGS Reacquisition")) {
```

Comment:
Confidence high!

Reply:
It passes the ITC "baseline" tests, so what could possibly go wrong?